File: myview/src/main/java/com/kingnet/myview/BaseActivity.java
package com.kingnet.myview;

import android.support.v7.app.AppCompatActivity;
import android.util.Log;

public class BaseActivity extends AppCompatActivity {

    @Override
    public void onWindowFocusChanged(boolean hasFocus) {
        Log.i("=============1",hasFocus+"");
        super.onWindowFocusChanged(hasFocus);
    }
}

File: wanandroid/src/main/java/com/mg/axechen/wanandroid/base/BaseActivity.kt
package com.mg.axechen.wanandroid.base

import android.support.v7.app.AppCompatActivity
import com.gyf.barlibrary.ImmersionBar

abstract class BaseActivity : AppCompatActivity() {

    private lateinit var immersionBar: ImmersionBar

    private var mCurrentTheme: Int = 0



}